ocs_ci/ocs/cluster.py

```diff
@@ -903,6 +904,88 @@ def get_osd_utilization():
     return osd_filled
 
 
+def get_ceph_df_detail():
+    """
+    Get ceph osd df detail
+
+    Returns:
+         dict: 'ceph df details' command output
+         
+    """
+    ceph_cmd="ceph df detail"
+    ct_pod = pod.get_ceph_tools_pod()
+    output = ct_pod.exec_ceph_cmd(ceph_cmd=ceph_cmd)
+    return output
+
+
+def validate_replica_data(pool_name, replica):
+    """
+    Check if data is replica 2 or 3
+
+    Args:
+        replica (int): size of the replica(2,3)
+        pool_name (str): name of the pool to check replica
+
+    Returns:
+        Bool: True if replicated data size is meet rep config and False if dont
+
+    """
+    from ocs_ci.ocs.exceptions import PoolNotFound
+    output = get_ceph_df_detail()
+    pool_list = output.get("pools")
+    for pool in pool_list:
+        if pool.get("name") == pool_name:
+            logger.info(f"{pool_name}")
+            stored = pool["stats"]["stored"]
+            byte_used = pool["stats"]["bytes_used"]
+            compress_bytes_used = pool["stats"]["compress_bytes_used"]
+            compress_under_bytes = pool["stats"]["compress_under_bytes"]
+            byte_used = byte_used + compress_under_bytes - compress_bytes_used
+            store_ratio = byte_used / stored
+            if (replica + 0.2) > store_ratio > (replica - 0.2):
+                logger.info(f"pool {pool_name} meet rep {replica} size")
+                return True
+            else:
+                logger.info(f"pool {pool_name} meet do not meet rep"
+                            f" {replica} size"
+                            f"Store ratio is {store_ratio}")
+                return False
```

Comment:
PEP8 standard:
You can change:

logger.info(f"pool {pool_name} meet do not meet rep"
                  f" {replica} size ,Store ratio is {store_ratio}"
)

Reply:
Does the rows need to be aligned by PEP8 standards? The IDE is marks it as not PEP8 standrad when I change the rows to be aligned like that.

Reply:
Do you mean only about the parentheses? Like it should be:
```logger.info(f"pool {pool_name} meet do not meet rep"
                  f" {replica} size ,Store ratio is {store_ratio}"
)```

Reply:
Do you mean only the parentheses in a new line?
like that?
```logger.info(f"pool {pool_name} meet do not meet rep"
                     f" {replica} size ,Store ratio is {store_ratio}"
)```

Reply:
Changed